spec/view_cooks_spec.rb

```diff
@@ -0,0 +1,48 @@
+require "rails_helper"
+
+feature "view cooks" do
+  scenario "user can see homepage" do
+    visit root_path
+    expect(page).to have_content "Batch-it!"
```

Comment:
Can we have some whitespace to indicate the different stages of the test?

(ditto throughout)

Reply:
Sure, thanks!